lemminx-maven/src/main/java/org/eclipse/lemminx/extensions/maven/searcher/watcher/LocalRepositoryFilesWatcher.java

```diff
@@ -0,0 +1,133 @@
+package org.eclipse.lemminx.extensions.maven.searcher.watcher;
+
+import java.io.IOException;
+import java.nio.file.DirectoryStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardWatchEventKinds;
+import java.nio.file.WatchEvent;
+import java.util.Collection;
+import java.util.Optional;
+
+import org.apache.maven.artifact.versioning.DefaultArtifactVersion;
+import org.eclipse.aether.artifact.Artifact;
+import org.eclipse.aether.artifact.DefaultArtifact;
+import org.eclipse.lemminx.extensions.maven.searcher.LocalRepositorySearcher;
+import org.eclipse.lsp4j.DidChangeWatchedFilesParams;
+
+/**
+ * This class provides the capability to track the deleted/created files from
+ * src/main/resources/templates folder of a Qute project and generates the LSP
+ * {@link DidChangeWatchedFilesParams} to update closed document cache.
+ * 
+ * This class is used only when the LSP client cannot support
+ * {@link DidChangeWatchedFilesParams}.
+ * 
+ * @author Angelo ZERR
+ *
+ */

```

Comment:
Please update the JavaDoc comment, as here this class has nothing to do with Qute templates

Reply:
fixed